nodejs/week2/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+/node_modules
```

Comment:
Hi Lucy

This should be `node_modules` instead of `/node_modules`. 

When you use `/node_modules` it refers to the `node_modules` folder in the same folder as the .gitignore, and none other. In this pull request there's a `node_modules` in the warmup folder that is not ignored. Could you fix this and make sure that no node_modules folder is part of the pull request? That way it'll be possible to review your changes ^_^

Thank you!

Reply:
Hi Jaime! Thank you, I have fixed node modules